src/chat.rs

```diff
@@ -1277,38 +1433,37 @@ pub fn marknoticed_all_chats(context: &Context) -> Result<(), Error> {
     sql::execute(
         context,
         &context.sql,
-        "UPDATE msgs    \
-         SET state=13 WHERE state=10;",
+        "UPDATE msgs
+            SET state=13
+          WHERE state=10;",
         params![],
     )?;
 
     context.call_cb(Event::MsgsChanged {
         msg_id: MsgId::new(0),
-        chat_id: 0,
+        chat_id: ChatId::new(0),
     });
 
     Ok(())
 }
 
 pub fn get_chat_media(
     context: &Context,
-    chat_id: u32,
+    chat_id: ChatId,
     msg_type: Viewtype,
     msg_type2: Viewtype,
     msg_type3: Viewtype,
 ) -> Vec<MsgId> {
     context
         .sql
         .query_map(
-            concat!(
-                "SELECT",
-                "    id",
-                " FROM msgs",
-                " WHERE chat_id=? AND (type=? OR type=? OR type=?)",
-                " ORDER BY timestamp, id;"
-            ),
+            "SELECT id
+               FROM msgs
+              WHERE chat_id=?
+                AND (type=? OR type=? OR type=?)
```

Comment:
Could have been `type IN (?,?,?)`

Reply:
Probably.  It's the type of change I'm trying to mix into this PR though.  It involves more tests etc.  I'll leave a comment.